### ISSUE TITLE
Crash app when I rotate my device

### ISSUE THREAD

**Issue by xmha97** [matdlu/OneTimeAlarm#9]
Hi, this app crashs when I rotate my device.
 
![Screenshot_2022-11-16-00-27-11-725_com.miui.bugreport.jpg](https://user-images.githubusercontent.com/23485114/202023912-e7ec460a-42fa-4b1c-976f-6b74dc3cbfd4.jpg)

**Comment by matdlu** [comment]
Yes I know of this issue. I will try to fix it when I have the time.

**Comment by xmha97** [comment]
Thanks.
